src/main/java/uk/gov/hmcts/reform/pip/model/enums/AuditAction.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.pip.model.enums;
+
+/**
+ * Enum that contains the different actions that are audited.
+ */
+public enum AuditAction {
```

Comment:
We also need one adding for Deleting a location

Reply:
Updated as discussed